packages/storybook-vue/src/stories/Textbox/TextboxWithFormBinding.stories.mdx

```diff
@@ -0,0 +1,47 @@
+import { action } from "@storybook/addon-actions";
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Textbox } from "@utrecht/component-library-vue";
+import { argTypes } from "./argTypes.ts";
+
+<Meta id="vue-textbox" title="Vue.js Component/Textbox/With Form Binding" component={Textbox} argTypes={argTypes} />
+
+# Textbox With Form Binding
+
+export const Template = (args, { argTypes }) => ({
+  props: Object.keys(argTypes),
+  components: { Textbox },
+  setup() {
+    return { args };
+  },
+  template: `
+  <FormField> 
+    <Textbox v-bind="args" @update:modelValue="action" :value="args.modelValue" />
+  <br/>
+  <br/>
+  <Paragraph v-if="args.modelValue"><strong> Textbox Binding:</strong> {{args.modelValue}}</Paragraph>
+  </FormField>
```

Comment:
These comments from the textarea PR also apply here:

- https://github.com/nl-design-system/utrecht/pull/1107/files#r935427944
- https://github.com/nl-design-system/utrecht/pull/1107/files#r935425726

Reply:
Regarding the second comment, I already [answered](https://github.com/nl-design-system/utrecht/pull/1107/files#r935510274), I need time to find a solution or maybe Storybook will release a new version that will solve this issue